minkindr/include/kindr/minimal/implementation/rotation-quaternion-inl.h

```diff
@@ -33,8 +33,10 @@ namespace minimal {
 
 template <typename Scalar>
 struct EPS {
-  static constexpr Scalar value();
-  static constexpr Scalar normalization_value();
+  static constexpr Scalar value() { return static_cast<Scalar>(1.0e-5); }
+  static constexpr Scalar normalization_value() {
+    return static_cast<Scalar>(1.0e-4);
```

Comment:
I have no idea who came up with the numbers or how they were chosen, I just moved they from being hard coded in each function to a single place. I also agree that some of them seem a bit odd.